pkg/harvester/detail/harvesterhci.io.virtualmachineimage/index.vue

```diff
@@ -57,6 +64,21 @@ export default {
       return this.value?.spec?.sourceType === 'upload';
     },
 
+    encryptionSecret() {
+      if (!this.value.isEncrypted) {
+        return '-';
```

Comment:
Just a nit thing, change to `&mdash;` character.

Reply:
I just followed other fields like description, url, errorMessage in this page using  '-'.

Anyway, this is just corner case since the image is not encrypted, it won't show encryptionSecret field.